src/App.js

```diff
@@ -1,24 +1,21 @@
-import logo from './logo.svg'
+import styled from 'styled-components'
+import emotions from './Data.json'
+import LearningCardPage from './pages/LearningCardPage'
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <AppWrapper emotions={emotions}>
```

Comment:
You don't need to pass `emotions` as props to your `AppWrapper`. It is only your styled-component and you don't use them anywhere.
```suggestion
    <AppWrapper>
```